File: development/sphinx-console-web/src/main/webapp/sphinx-console/app/model/SearchQueryGrouped.js
Ext.define('sphinx-console.model.SearchQueryGrouped', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'id'},
        {name: 'query'},
        {name: 'collectionName'},
        {name: 'replicaName'},
        {name: 'replicaNameStr',
            calculate: function (data) {
                return data.replicaName ? data.replicaName : "-";
            }
        },
        {name: 'totalTimeMin'},
        {name: 'totalTimeMax'},
        {name: 'time',
            calculate: function (data) {
                return data.totalTimeMin == data.totalTimeMax ? data.totalTimeMin : data.totalTimeMin + " - " + data.totalTimeMax;
            }
        },
        {name: 'searchQueryResultCount'},
        {name: 'resultCountMin'},
        {name: 'resultCountMax'},
        {name: 'resultCount',
            calculate: function (data) {
                return data.resultCountMin == data.resultCountMax ? data.resultCountMin : data.resultCountMin + " - " + data.resultCountMax;
            }
        },
        {name: 'offsetNotZeroCount'}
    ]
});

File: development/sphinx-console-base/sphinx-console-common-model/src/main/java/ru/skuptsov/sphinx/console/coordinator/model/search/query/SearchQueryGrouped.java
package ru.skuptsov.sphinx.console.coordinator.model.search.query;

public class SearchQueryGrouped {

    private Long searchQueryId;
    private String query;
    private String collectionName;
    private String replicaName;
    private Integer totalTimeMin;
    private Integer totalTimeMax;
    private Integer resultCountMin;
    private Integer resultCountMax;
    private Integer offsetNotZeroCount;
    private Integer searchQueryResultCount;

    public SearchQueryGrouped() {
    }

    public SearchQueryGrouped(Long searchQueryId, String query, String collectionName, Integer totalTimeMin, Integer totalTimeMax, Integer resultCountMin, Integer resultCountMax, Integer offsetNotZeroCount, Integer searchQueryResultCount) {
        this.searchQueryId = searchQueryId;
        this.query = query;
        this.collectionName = collectionName;
        this.totalTimeMin = totalTimeMin;
        this.totalTimeMax = totalTimeMax;
        this.resultCountMin = resultCountMin;
        this.resultCountMax = resultCountMax;
        this.offsetNotZeroCount = offsetNotZeroCount;
        this.searchQueryResultCount = searchQueryResultCount;
    }

    public Long getSearchQueryId() {
        return searchQueryId;
    }

    public void setSearchQueryId(Long searchQueryId) {
        this.searchQueryId = searchQueryId;
    }

    public String getQuery() {
        return query;
    }

    public void setQuery(String query) {
        this.query = query;
    }

    public String getCollectionName() {
        return collectionName;
    }

    public void setCollectionName(String collectionName) {
        this.collectionName = collectionName;
    }

    public String getReplicaName() {
        return replicaName;
    }

    public void setReplicaName(String replicaName) {
        this.replicaName = replicaName;
    }

    public Integer getTotalTimeMin() {
        return totalTimeMin;
    }

    public void setTotalTimeMin(Integer totalTimeMin) {
        this.totalTimeMin = totalTimeMin;
    }

    public Integer getTotalTimeMax() {
        return totalTimeMax;
    }

    public void setTotalTimeMax(Integer totalTimeMax) {
        this.totalTimeMax = totalTimeMax;
    }

    public Integer getResultCountMin() {
        return resultCountMin;
    }

    public void setResultCountMin(Integer resultCountMin) {
        this.resultCountMin = resultCountMin;
    }

    public Integer getResultCountMax() {
        return resultCountMax;
    }

    public void setResultCountMax(Integer resultCountMax) {
        this.resultCountMax = resultCountMax;
    }

    public Integer getOffsetNotZeroCount() {
        return offsetNotZeroCount;
    }

    public void setOffsetNotZeroCount(Integer offsetNotZeroCount) {
        this.offsetNotZeroCount = offsetNotZeroCount;
    }

    public Integer getSearchQueryResultCount() {
        return searchQueryResultCount;
    }

    public void setSearchQueryResultCount(Integer searchQueryResultCount) {
        this.searchQueryResultCount = searchQueryResultCount;
    }
}
